py/mattak/backends/uproot/dataset.py

```diff
@@ -320,23 +321,23 @@ def eventInfo(self, override_skip_incomplete : Optional[bool] = None) -> Union[O
 
         infos = []
         info = None  # if range(0)
-        for i in range(self.last - self.first):
+        for i, triggerInfoi in zip(range(self.last - self.first), triggerInfo):
```

Comment:
can we use a different name?

Reply:
it's an iterated variable - I'll accept any name suggestions